### ISSUE TITLE
Doesn't work with uBlock Origin

### ISSUE THREAD

**Issue by CodeNeedsCoffee** [badsgahhl/pihole-browser-extension#83]
When you have both uBlock Origin and the PiHole Extension installed, the PiHole Extension stops working.
![image](https://github.com/badsgahhl/pihole-browser-extension/assets/16927633/420f63fd-3921-4e10-b745-2ac391f3321a)

Installed PiHole from here:
https://microsoftedge.microsoft.com/addons/detail/pihole-browser-extension/jbelapanpabbnchmaalboimjmgmbfjnm

Installed uBlock Origin from here:
https://chrome.google.com/webstore/detail/ublock-origin-lite/ddkjiahejlhfcafbddmgiahcphecmpfh

 - OS: Windows 11
 - Browser Edge
 - Version 114.0.1823.58

**Comment by drewfatstacks** [comment]
Works for me.  Windows 10, Chrome 115.0.5790.173, uBlock 1.51.0
Same for Firefox 116.0.2 and uBlock 1.51.0
I use the Pihole Switch to control two Piholes

**Comment by CodeNeedsCoffee** [comment]
Super odd error/bug. I noticed if I put the IP address or if I put the pi.hole local name, both ended up failing. I did put it behind my ngnix proxy and as soon as I added an SSL cert, it started working.

To add to the confusion, I know my key was correct because it would green "Connection successful" dialog at the bottom. Additionally, it would appear to work at first when I would go to a site, but as soon as I tried to interact with the browser extension popup (whatever that part is called) it would then go into an error state.

Either way, it's working now!
